serverjs/cubefn.js

```diff
@@ -228,6 +228,48 @@ async function getElo(cardnames, round) {
   return result;
 }
 
+function addDraftFormat(cube, { title, multiples, html, packs }) {
```

Comment:
Could we add tests for these new functions?

Reply:
Yup can do.
Can I get some feedback first that this is the style that we want and this is the place where they should live? 
I'm used to CRUD functions being attached to controllers/models, which we don't have, so this is something like that but less elegant.
Also using some newfangled js style here, with destructuring the params and but I think it reads pretty well. Thoughts?

Reply:
Tests added.